skbio/sequence/_rna.py

```diff
@@ -135,6 +135,73 @@ def degenerate_map(cls):
     def _motifs(self):
         return _motifs
 
+    @stable(as_of="0.4.0-dev")
+    def reverse_transcribe(self):
+        """Reverse transcribe RNA into DNA.
+
+        RNA sequence is assumed to be the coding strand. Uracil (U) is
```

Comment:
actually we probably use the this terminology (
https://en.wikipedia.org/wiki/Coding_strand):

Wherever a gene <https://en.wikipedia.org/wiki/Gene> exists on a DNA
molecule <https://en.wikipedia.org/wiki/Molecule>, one strand is the coding
strand (or *sense strand* or *non-template strand*), and the other is
the *noncoding
strand* (also called the antisense
<https://en.wikipedia.org/wiki/Sense_%28molecular_biology%29> strand,[1]
<https://en.wikipedia.org/wiki/Coding_strand#cite_note-Lewin08-1>
anticoding strand, template strand, or transcribed strand).

I prefer the use of "sense" and "antisense" to accommandate both
protein-coding and non-protein-coding genes...

On Mon, Aug 17, 2015 at 2:49 PM, Evan Bolyen <notifications@github.com>
wrote:

> In skbio/sequence/_rna.py
> <https://github.com/biocore/scikit-bio/pull/1057#discussion_r37241162>:
>
> > @@ -135,6 +135,73 @@ def degenerate_map(cls):
> >      def _motifs(self):
> >          return _motifs
> >
> > +    @stable(as_of="0.4.0-dev")
> > +    def reverse_transcribe(self):
> > +        """Reverse transcribe RNA into DNA.
> > +
> > +        RNA sequence is assumed to be the coding strand. Uracil (U) is
>
> I like the first suggestion.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/scikit-bio/pull/1057/files#r37241162>.
>